src/module.rs

```diff
@@ -1073,6 +1092,176 @@ impl DataLayout {
         }
         data_layout
     }
+
+    /// Port of `llvm::DataLayout::getTypeStoreSize`.
+    pub fn get_type_store_size(&self, types: &Types, ty: &Type) -> Option<TypeSize> {
+        let base_size = self.get_type_size_in_bits(types, ty)?;
+        fn divide_ciel(numerator: u64, denominator: u64) -> u64 {
+            let aligned = align_to_abi(
+                numerator,
+                &Alignment {
+                    abi: u32::try_from(denominator).unwrap(),
+                    pref: 0,
+                },
+            );
+            aligned / denominator
+        }
+        Some(TypeSize::new(
+            divide_ciel(base_size.quantity, 8),
```

Comment:
where does the `8` come from?  is it bits-in-byte?  in that case could we instead have a `get_type_size_in_bytes()` method and have the division happen there?